opus/application/apps/ui/templates/ui/header.html

```diff
@@ -59,6 +59,7 @@
         const DEFAULT_SORT_ORDER = '{{ default_sort_order }}';
         const STATIC_URL = '{{ STATIC_URL }}';
         const MAX_SELECTIONS_FOR_DATA_DOWNLOAD = '{{ MAX_SELECTIONS_FOR_DATA_DOWNLOAD }}';
+        const MAX_SELECTIONS_ALLOWED = '{{ max_selections_allowed }}';
```

Comment:
The line about "MAX_SELECTIONS_FOR_DATA_DOWNLOAD" seems to be old, dead code. The variable isn't even being passed in to the template context. Let's go ahead and remove it.